btf_types/src/lib.rs

```diff
@@ -152,6 +153,39 @@ impl BrainFuckProgram {
         let bf_program = Self::new(file_path_ref, content);
         Ok(bf_program)
     }
+
+    /// Validate if brackets are balanced.
+    pub fn validate_brackets(&self) -> Result<(), String> {
```

Comment:
Will this function work for the following input?  `][`

Reply:
Yeah, you are right.
I have added one more check for that